core/embed/models/T3W1/boards/trezor_t3w1_revA.h

```diff
@@ -0,0 +1,101 @@
+#define I2C_INSTANCE_0_SCL_PIN GPIO_PIN_0
+#define I2C_INSTANCE_0_SCL_CLK_EN __HAL_RCC_GPIOC_CLK_ENABLE
+#define I2C_INSTANCE_0_RESET_REG &RCC->APB3RSTR
+#define I2C_INSTANCE_0_RESET_BIT RCC_APB3RSTR_I2C3RST
+#define I2C_INSTANCE_0_EV_IRQHandler I2C3_EV_IRQHandler
+#define I2C_INSTANCE_0_ER_IRQHandler I2C3_ER_IRQHandler
+#define I2C_INSTANCE_0_EV_IRQn I2C3_EV_IRQn
+#define I2C_INSTANCE_0_ER_IRQn I2C3_ER_IRQn
+#define I2C_INSTANCE_0_GUARD_TIME 0
+
+#define I2C_INSTANCE_1 I2C4
+#define I2C_INSTANCE_1_CLK_EN __HAL_RCC_I2C4_CLK_ENABLE
+#define I2C_INSTANCE_1_CLK_DIS __HAL_RCC_I2C4_CLK_DISABLE
+#define I2C_INSTANCE_1_PIN_AF GPIO_AF4_I2C4
+#define I2C_INSTANCE_1_SDA_PORT GPIOD
+#define I2C_INSTANCE_1_SDA_PIN GPIO_PIN_13
+#define I2C_INSTANCE_1_SDA_CLK_EN __HAL_RCC_GPIOD_CLK_ENABLE
+#define I2C_INSTANCE_1_SCL_PORT GPIOD
+#define I2C_INSTANCE_1_SCL_PIN GPIO_PIN_12
+#define I2C_INSTANCE_1_SCL_CLK_EN __HAL_RCC_GPIOD_CLK_ENABLE
+#define I2C_INSTANCE_1_RESET_REG &RCC->APB1RSTR2
+#define I2C_INSTANCE_1_RESET_BIT RCC_APB1RSTR2_I2C4RST
+#define I2C_INSTANCE_1_EV_IRQHandler I2C4_EV_IRQHandler
+#define I2C_INSTANCE_1_ER_IRQHandler I2C4_ER_IRQHandler
+#define I2C_INSTANCE_1_EV_IRQn I2C4_EV_IRQn
+#define I2C_INSTANCE_1_ER_IRQn I2C4_ER_IRQn
+#define I2C_INSTANCE_1_GUARD_TIME 0
+
+#define I2C_INSTANCE_2 I2C2
+#define I2C_INSTANCE_2_CLK_EN __HAL_RCC_I2C2_CLK_ENABLE
+#define I2C_INSTANCE_2_CLK_DIS __HAL_RCC_I2C2_CLK_DISABLE
+#define I2C_INSTANCE_2_PIN_AF GPIO_AF4_I2C2
+#define I2C_INSTANCE_2_SDA_PORT GPIOF
+#define I2C_INSTANCE_2_SDA_PIN GPIO_PIN_0
+#define I2C_INSTANCE_2_SDA_CLK_EN __HAL_RCC_GPIOF_CLK_ENABLE
+#define I2C_INSTANCE_2_SCL_PORT GPIOF
+#define I2C_INSTANCE_2_SCL_PIN GPIO_PIN_1
+#define I2C_INSTANCE_2_SCL_CLK_EN __HAL_RCC_GPIOF_CLK_ENABLE
+#define I2C_INSTANCE_2_RESET_REG &RCC->APB1RSTR1
+#define I2C_INSTANCE_2_RESET_BIT RCC_APB1RSTR1_I2C2RST
+#define I2C_INSTANCE_2_EV_IRQHandler I2C2_EV_IRQHandler
+#define I2C_INSTANCE_2_ER_IRQHandler I2C2_ER_IRQHandler
+#define I2C_INSTANCE_2_EV_IRQn I2C2_EV_IRQn
+#define I2C_INSTANCE_2_ER_IRQn I2C2_ER_IRQn
+#define I2C_INSTANCE_2_GUARD_TIME 0
+
+#define I2C_INSTANCE_3 I2C1
+#define I2C_INSTANCE_3_CLK_EN __HAL_RCC_I2C1_CLK_ENABLE
+#define I2C_INSTANCE_3_CLK_DIS __HAL_RCC_I2C1_CLK_DISABLE
+#define I2C_INSTANCE_3_PIN_AF GPIO_AF4_I2C1
+#define I2C_INSTANCE_3_SDA_PORT GPIOG
+#define I2C_INSTANCE_3_SDA_PIN GPIO_PIN_13
+#define I2C_INSTANCE_3_SDA_CLK_EN __HAL_RCC_GPIOG_CLK_ENABLE
+#define I2C_INSTANCE_3_SCL_PORT GPIOG
+#define I2C_INSTANCE_3_SCL_PIN GPIO_PIN_14
+#define I2C_INSTANCE_3_SCL_CLK_EN __HAL_RCC_GPIOG_CLK_ENABLE
+#define I2C_INSTANCE_3_RESET_REG &RCC->APB1RSTR1
+#define I2C_INSTANCE_3_RESET_BIT RCC_APB1RSTR1_I2C1RST
+#define I2C_INSTANCE_3_EV_IRQHandler I2C1_EV_IRQHandler
+#define I2C_INSTANCE_3_ER_IRQHandler I2C1_ER_IRQHandler
+#define I2C_INSTANCE_3_EV_IRQn I2C1_EV_IRQn
+#define I2C_INSTANCE_3_ER_IRQn I2C1_ER_IRQn
+#define I2C_INSTANCE_3_GUARD_TIME 0
+
+#define TOUCH_SENSITIVITY 0x40
```

Comment:
The touch control used on the T3W1 has no sensitivity register (like the FT36x6). It's a minor problem for now, but we should address it later.